app/src/visionglass/res/values/themes.xml

```diff
@@ -15,6 +15,156 @@
+        <item name="cornerSize">32dp</item>
+    </style>
+
+    <style name="ThemeOverlay.App.Slider" parent="">
+        <item name="colorPrimary">@color/white</item>
+        <item name="colorOnPrimary">@color/white</item>
+    </style>
+
+    <style name="BaseToolbarButton" parent="Widget.MaterialComponents.Button">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:backgroundTint">@color/bg_button_checkable</item>
+        <item name="android:scaleType">fitCenter</item>
+        <item name="android:layout_margin">8dp</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+        <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.App.Button.Rounded</item>"
+    </style>
+
+    <style name="BaseFooterButton" parent="Widget.MaterialComponents.Button">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:scaleType">fitCenter</item>
+        <item name="android:backgroundTint">@color/asphalt</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+        <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.App.Button.Rounded</item>"
+    </style>
+
+    <style name="BaseBorderlessButton" parent="Widget.AppCompat.Button.Borderless">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">match_parent</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+    </style>
+
+    <style name="BaseMediaButton" parent="BaseBorderlessButton">
+        <item name="android:layout_weight">1</item>
+    </style>
+
+
+    <!-- Specific Styles -->
+    <style name="ToolbarStyle" parent="Widget.MaterialComponents.Toolbar.Primary">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:elevation">0dp</item>
+        <item name="android:paddingStart">8dp</item>
+        <item name="android:paddingEnd">8dp</item>
+        <item name="logoAdjustViewBounds">true</item>
+        <item name="navigationIcon">@drawable/ff_logo_icon_48</item>
+        <item name="popupTheme">@style/Theme.WolvicPhone.PopupOverlay</item>
+    </style>
+
+    <style name="HeadlockToggleButtonStyle" parent="BaseToolbarButton">
+        <item name="icon">@drawable/headlock_toggle_button_states</item>
+        <item name="android:checkable">true</item>
+        <item name="android:checked">false</item>
+        <item name="layout_constraintEnd_toStartOf">@id/realign_button</item>
```

Comment:
Constraints define how the widget fits in a particular layout, so they should remain in the layout file.